test/ssl_tests.cpp

```diff
@@ -26,8 +31,7 @@ TEST(SslTests, HelloWorldTestSimpel) {
     std::this_thread::sleep_for(std::chrono::seconds(1));
 
     Url url{server->GetBaseUrl() + "/hello.html"};
-    std::string baseDirPath = server->getBaseDirPath();
-    SslOptions sslOpts = Ssl(ssl::CaPath{baseDirPath + "ca.cer"}, ssl::CertFile{baseDirPath + "client.cer"}, ssl::KeyFile{baseDirPath + "client.key"}, ssl::VerifyPeer{false}, ssl::PinnedPublicKey{baseDirPath + "server.pubkey"}, ssl::VerifyHost{false}, ssl::VerifyStatus{false});
+    SslOptions sslOpts = Ssl(ssl::CaPath{std::move(caCertPath)}, ssl::CertFile{std::move(clientCertPath)}, ssl::KeyFile{std::move(clientKeyPath)}, ssl::VerifyPeer{false}, ssl::PinnedPublicKey{std::move(serverPubKeyPath)}, ssl::VerifyHost{false}, ssl::VerifyStatus{false});
```

Comment:
`std::move(...)` with static variables is no good code style.

Reply:
I replaced all occurrences of std::move on static variables by string-concatenations for generating rvalue references in-place, similar to the way it was before. I removed the unused static variables that I introduced. This should resolve the issue